### ISSUE TITLE
Error after launching PlanetGeneratorUI.exe

### ISSUE THREAD

**Issue by xan2622** [giogix2/PlanetGenerator#8]
Hello.

I use Windows 8.1 64bits and I couldn't launch PlanetGeneratorUI.exe.
Here is the error that I got : 

![screenshot001](https://cloud.githubusercontent.com/assets/1732773/15268756/aefa224e-19e9-11e6-8d39-fe0565ff8d9b.png)

Thanks

**Comment by LiPingjiang** [comment]
Have you try the solution:
http://stackoverflow.com/questions/20495620/qt-5-1-1-application-failed-to-start-because-platform-plugin-windows-is-missi 

Try to put the DLL file in the root folder?

[libEGL.zip](https://github.com/giogix2/PlanetGenerator/files/264918/libEGL.zip)

**Comment by giogix2** [comment]
That problem might be related with the fact that you're using a 64 bit platform. I actually had the same problem when I moved to Windows 10.
Another reason could be that you are using the DirectX 12, which is not compatible with Ogre 3D (at least Ogre 3D 1.9).
I built another version of the software for Windows 64 bit, which uses OpenGL instead of DirectX.
You can find it in the releases section. Anyway I will attach the link here: https://github.com/giogix2/PlanetGenerator/releases/download/v0.2_64bit/PlanetGenerator_64bit.rar
Thanks for reporting the issue.

**Comment by giogix2** [comment]
Actually I found out that the problem wasn't related only on the platform (32 bit or 64 bit) or the rendering system (DirectX or OpenGL). I tried to launch the software in another computer, and the same error reported by @x-a-n-a-x  showed up.
The error occurs when Qt is not installed in the system.
To solve this problem the solution is reported here: http://stackoverflow.com/questions/16773789/qt5-static-build-yields-failed-to-load-platform-plugin-windows
Basically it was necessary to copy the folder containing the file "qwindows.dll" from Qt to the directory with the executable of PlanetGenerator (the folder is called "platforms").

I put the required folder with the executable and it should work now. The download link is still the same as in the previous post.
